DS/DSAssignmrnt8/TreeSort.java

```diff
@@ -0,0 +1,20 @@
+public class TreeSort {
+	
+
+	private Tree createTree(int rollNo[]) {
+		Tree tree= new Tree();	
+		for (int i = 0; i < rollNo.length; i++) {
+
+			tree.addNode(rollNo[i]);
+
+		}
+   return tree;
+	}
+
+	public int[] treeSort(int rollNo[]) {
+		Tree tree=createTree(rollNo);
+		int output[] = tree.inOrder(tree.getRoot());
```

Comment:
You can skip assigning this to a variable which is only used to return. You can directly write:- 
    
    return tree.inOrder(tree.getRoot());

This will not hurt readability